activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/ManualTaskConverterTest.java

```diff
@@ -0,0 +1,51 @@
+package org.activiti.editor.language.xml;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.activiti.bpmn.converter.BpmnXMLConverter;
+import org.activiti.bpmn.model.BpmnModel;
+import org.activiti.bpmn.model.FlowElement;
+import org.activiti.bpmn.model.ManualTask;
+import org.junit.Test;
+
+public class ManualTaskConverterTest extends AbstractConverterTest {
+
+    @Test
+    public void convertXMLToModel() throws Exception {
+        BpmnModel bpmnModel = readXMLFile();
+        validateModel(bpmnModel);
+    }
+
+    @Test
+    public void convertModelToXML() throws Exception {
+        BpmnModel bpmnModel = readXMLFile();
+        BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
+        validateModel(parsedModel);
+        deployProcess(parsedModel);
+    }
+
+    @Override
+    protected String getResource() {
+        return "manualtaskmodel.bpmn";
```

Comment:
Please, use the camel case in the file names.

Reply:
fixed